config/nrfconnect/chip-module/Kconfig.features

```diff
@@ -73,6 +73,21 @@ config SPI_NOR_FLASH_LAYOUT_PAGE_SIZE
 
 endif # CHIP_SPI_NOR
 
+config CHIP_MEMORY_PROFILING
+	bool "Enable features for tracking memory usage"
+	default n
```

Comment:
nit: default n is redundant: https://docs.zephyrproject.org/latest/build/kconfig/tips.html#id20